Analysis/Core/src/CorrelationContainer.cxx

```diff
@@ -331,7 +331,6 @@ TString CorrelationContainer::combineBinning(TString defaultBinning, TString cus
       LOGF(info, "Using custom binning for %s", tag.Data());
     }
   }
-  delete lines;
   binningStr += customBinning;
```

Comment:
I don't mind your change, but here there was no mem leak. I was properly cleaning the TObjArray. Or am I missing something?

Reply:
At first I decided to change everything to `unique_ptr` for uniformity, but I think I will fix only real leaks in tasks (`.cxx`-files) and won't touch test macros (`.C` files). I'll force-push a smaller commit instead.

Reply:
For me your change is fine, really! I just wanted to avoid the impression that there was a leak in the code.